### ISSUE TITLE
Enhancement: Create a "claim" button for reviewing summaries

### ISSUE THREAD

**Issue by skeptic17** [newsvoice/newsvoice#237]
Reviewing a summary can take a while if you want to check facts in the stories. I have had other people approve summaries that I was working on (and, quite likely, rejecting). This discourages/undermines careful review. There should be a "claim" button for summaries to avoid this.

**Comment by Digiguy** [comment]
Agreed Skeptic. I saw some pretty shakey claims of being "a story" calling themselves news articles. Click bait at best.. 
It would help if when one submits a story they have to include (for politics related) sources from right and left for a submission.
There are some questionable sources as well. Dailymail.uk comes to mind...
There are quite a few 'articles' that are not articles, but 2 or 3 paragraphs with no sources other than instagram posts that make them not really a news story, but hearsay.
